integration-tests/src/test/java/com/redhat/service/smartevents/integration/tests/steps/Hooks.java

```diff
@@ -88,7 +89,7 @@ public void cleanUp() {
                                 Awaitility.await()
                                         .atMost(Duration.ofMinutes(4))
                                         .pollInterval(Duration.ofSeconds(5))
-                                        .until(() -> BridgeResource.getBridgeList(token).getItems().stream().noneMatch(b -> b.getId().equals(bridgeId)));
+                                        .untilAsserted(() -> assertThat(BridgeResource.getBridgeList(token).getItems()).noneMatch(b -> b.getId().equals(bridgeId)));
```

Comment:
similar comment as above

Reply:
Could we also please use `Objects.euqals(a, b)` instead of `a.equals(b)`. The first one should be safer from null pointer exceptions.